### PR TITLE
Array functions removal

### PR DESCRIPTION
**Pull request by euryalus**
There where some unnecessary conversions (np.array and np.asarray) that were deleted.

### CODE REVIEW

File: pye3d/detector_3d.py [hunk 1]
```diff
@@ -334,7 +334,7 @@ def _predict_from_3d_search(
             _EYE_RADIUS_DEFAULT,
             self.camera.focal_length,
             self.camera.resolution,
-            major_axis_factor=2.5,
```
Comment (papr) [inline pye3d/detector_3d.py:335]: I feel like this should be in a separate commit. What does it change?

**Review by papr — COMMENTED** [review]
I think it would be nice to rebase this PR onto `master` to clean up its commit history.

File: pye3d/detector_3d.py (hunk 1 above)
Comment (euryalus) [reply pye3d/detector_3d.py:335]: Yes, true. This sets the size of the crop for the 3d_search. Had forgotten that I had changed this for testing purposes.

**Review by papr — APPROVED** [review]
